static/src/javascripts/projects/common/modules/ui/notification-counter.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import mediator from 'lib/mediator';
+
+const ORIGINAL_PAGETITLE = document.title;
```

Comment:
Can we move this assignment into `initNotificationCounter` and pass `ORIGINAL_PAGETITLE` into `setCount`? This would remove the side effect of loading the module